### PR TITLE
Decode Remote Google Logs

### PR DESCRIPTION
**Pull request by KevYuen**
The `Blob.download_as_string` function returns a byte which cause
the log result to be displayed in a single line like:

b"line1\nline2"

instead of

line1
line2

added an isinstance check to make sure it doesn't break if it
returns string in some case and not others

all the logs is in a single line: 

image

Version: 2.0.0rc3

TODO: 
---
- [x] Update test to include case with byte return value instead of string (apologies, fighting with breeze)

### CODE REVIEW

File: airflow/providers/google/cloud/log/gcs_task_handler.py [hunk 1]
```diff
@@ -153,6 +153,10 @@ def _read(self, ti, try_number, metadata=None):
         try:
             blob = storage.Blob.from_string(remote_loc, self.client)
             remote_log = blob.download_as_string()
+
+            if isinstance(remote_log, bytes):
+                remote_log = remote_log.decode()
```
Comment (turbaszek) [inline airflow/providers/google/cloud/log/gcs_task_handler.py]: I would say that this should be part of `download_as_string` method. The name suggest that the output should be a `str` not `bytes` so fixing it there may solve some issues in future.

Comment (KevYuen) [reply airflow/providers/google/cloud/log/gcs_task_handler.py]: You're right, looks like the method is deprecated and it is now an alias for `download_as_byte`

https://googleapis.dev/python/storage/latest/blobs.html#google.cloud.storage.blob.Blob.download_as_string

Will update the PR

**Review by turbaszek — APPROVED** [review]

**Comment by KevYuen** [comment]
@turbaszek Is there anything I should do to get this PR merged? Apologies, I am not too familiar with the process.

**Review by turbaszek — APPROVED** [review]

**Comment by turbaszek** [comment]
@KevYuen pleas take a look at the pylint issues on CI

**Comment by KevYuen** [comment]
@turbaszek I missed that! Thanks 👍

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Thanks @KevYuen !

**Comment by brenocosta0901** [comment]
@KevYuen @potiuk I would like to know if these changes will enter on the next Airflow version. I'm facing this problem when migrating to version 2.0.1.

**Comment by potiuk** [comment]
Those changes are available even now in already released 2.0.0 version of  Google providers - see the changelog: https://airflow.apache.org/docs/apache-airflow-providers-google/2.0.0/index.html 

You can upgrade even now to the new google providers without waiting for the new Airflow version (which will include it by default).

**Pull request merged**